README.md

```diff
@@ -89,8 +89,18 @@ data.main.temp # => 12
 Returns weather by city, optional state (in the US) and optional ISO 3166 country code.
 
 ```ruby
-client.current_city('New York', 'NY', 'US')
-client.current_weather(city: 'New York', state: 'NY', country: 'US')
+client.current_city('Sydney')
+client.current_city('London, UK')
+client.current_city('London', 'UK')
+client.current_city('Albany')
+client.current_city('Albany, New York')
+client.current_city('Albany, New York', 'US')
+client.current_city('Albany, NY', 'US')
+client.current_city('Albany', 'New York', 'US')
+client.current_city('Albany', 'NY', 'US')
+client.current_city('Albany', 'NY')  # Bad: 2 letter state abbreviation without country will give Faraday::Resource not found
```

Comment:
Say "2-letter state abbreviation w/o country will raise Faraday::ResourceNotFound" (this is the correct spelling of the error)